.editorconfig

```diff
@@ -0,0 +1,28 @@
+# EditorConfig for MakerDAO Projects
+
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
```

Comment:
```suggestion
charset = utf-8
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
```

Reply:
Just fixed in https://github.com/brianmcmichael/mcd-editorconfig/blob/master/.editorconfig

Reply:
Shouldn't we fix this in this repo as well?